src/MoleculeLookup.h

```diff
@@ -29,22 +29,17 @@ class MoleculeLookup
 {
 public:
 
-  MoleculeLookup() : molLookup(NULL), boxAndKindStart(NULL), boxAndKindSwappableCounts(NULL),
-   molIndex(NULL), atomIndex(NULL), molKind(NULL), atomKind(NULL), atomCharge(NULL) {}
+  MoleculeLookup(){}
 
   ~MoleculeLookup()
-  {
-    delete[] molLookup;
-    delete[] boxAndKindStart;
-    delete[] molIndex;
-    delete[] atomIndex;
-    delete[] molKind;
-    delete[] atomKind;
-    delete[] atomCharge;
-    delete[] boxAndKindSwappableCounts;
-    delete[] originalMoleculeIndices;
-    delete[] permutedMoleculeIndices;
-  }
```

Comment:
Why dont we deallocate the memory in destructor?

Reply:
It's now a vector.

Reply:
Restored these to dynamically allocated arrays.  I used the binary_data() serialization function of Cereal, so I don't use vectors here at all.